### PR TITLE
adding Horusec

### PR DESCRIPTION
**Pull request by caroldf07**
I think my project would be a great fit for Up for Grabs.

Let me know if there's any feedback you have about the template!

### CODE REVIEW

File: _data/projects/horusec.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - typescript
+  - go
+  - pipeline
+  - SAST
+  - CI/CD
+upforgrabs:
+  name: good first issue
+  link: https://github.com/ZupIT/horusec/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
+stats:
+  issue-count:
```
Comment (ritwik12) [inline _data/projects/horusec.yml]: You don't need to put these, It gets fetched and appended automatically later.

File: _data/projects/horusec.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - php
+  - kotlin
+  - typescript
+  - go
+  - pipeline
+  - SAST
+  - CI/CD
+upforgrabs:
+  name: good first issue
+  link: https://github.com/ZupIT/horusec/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```
Comment (ritwik12) [inline _data/projects/horusec.yml]: Please change this to https://github.com/ZupIT/horusec/labels/good%20first%20issue

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by ritwik12** [comment]
@caroldf07 Reviewed all your PRs, please make changes. 

Saw your profile. Developer Advocate kinda looks fancy but not sure what that role is. Is it a mix of Developer and Partner engineer types, Consultanting and all?

**Review by ritwik12 — APPROVED** [review]

**Comment by caroldf07** [comment]
> @caroldf07 Reviewed all your PRs, please make changes.
> 
> Saw your profile. Developer Advocate kinda looks fancy but not sure what that role is. Is it a mix of Developer and Partner engineer types, Consultanting and all?

Kinda of, kkkk. I'm responsible for make comunity to knows about Zup's open source projects, talk about and receive feeback about them to improve developer's experience using our solutions :)

**Comment by ritwik12** [comment]
> > @caroldf07 Reviewed all your PRs, please make changes.
> > Saw your profile. Developer Advocate kinda looks fancy but not sure what that role is. Is it a mix of Developer and Partner engineer types, Consultanting and all?
> 
> Kinda of, kkkk. I'm responsible for make comunity to knows about Zup's open source projects, talk about and receive feeback about them to improve developer's experience using our solutions :)

That's cool, will read blogs on that then, thanks

**Comment by ritwik12** [comment]
@caroldf07 CI is falling for all your PRs, please check.

**Comment by ritwik12** [comment]
@caroldf07 I guess you pulled and merged. please revert that back to your original yml. CI is failing, will merge your PR once #3064 is solved.

**Comment by ritwik12** [comment]
@caroldf07 Please refresh pull once and rebase

File: _data/projects/horusec.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  - java
+  - javascript
+  - security
+  - php
+  - kotlin
+  - typescript
+  - go
+  - pipeline
+  - SAST
+  - CI/CD
```
Comment (ritwik12) [inline _data/projects/horusec.yml]: @caroldf07 We need to fix these errors, please change names accordingly.

- Tag 'SAST' contains invalid characters. Allowed characters: a-z, 0-9, +, #, . or -
- Tag 'CI/CD' contains invalid characters. Allowed characters: a-z, 0-9, +, #, . or -

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by ritwik12** [comment]
@caroldf07 Please follow the same tag names in your other PRs too.

**Review by ritwik12 — APPROVED** [review]

File: _data/projects/horusec.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  - javascript
+  - security
+  - php
+  - kotlin
+  - typescript
+  - go
+  - pipeline
+  - sast
+  - continuous integrations
+  - continuous delivery
```
Comment (ritwik12) [inline _data/projects/horusec.yml]:
```suggestion
  - continuous-integrations
  - continuous-delivery
```

**Review by ritwik12 — CHANGES_REQUESTED** [review]

File: _data/projects/horusec.yml (hunk 4 above)
Comment (caroldf07) [reply _data/projects/horusec.yml]: I saw it, I've just removed

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@caroldf07 Thanks!!

**Pull request merged**